[2021-08-12 20:57:27,418] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T20:57:19.032000+00:00 [queued]>
[2021-08-12 20:57:27,444] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T20:57:19.032000+00:00 [queued]>
[2021-08-12 20:57:27,445] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 20:57:27,447] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-08-12 20:57:27,448] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 20:57:27,459] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): thursday_Task> on 2021-08-12T20:57:19.032000+00:00
[2021-08-12 20:57:27,471] {standard_task_runner.py:52} INFO - Started process 2753 to run task
[2021-08-12 20:57:27,477] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pipeline_try', 'thursday_Task', '2021-08-12T20:57:19.032000+00:00', '--job-id', '120', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_try1.py', '--cfg-path', '/tmp/tmpdop8qiuv', '--error-file', '/tmp/tmpyyxablcm']
[2021-08-12 20:57:27,484] {standard_task_runner.py:77} INFO - Job 120: Subtask thursday_Task
[2021-08-12 20:57:27,550] {logging_mixin.py:104} INFO - Running <TaskInstance: pipeline_try.thursday_Task 2021-08-12T20:57:19.032000+00:00 [running]> on host 99300daf4d26
[2021-08-12 20:57:27,624] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=ameysonje242000@gmail.com
AIRFLOW_CTX_DAG_OWNER=amey
AIRFLOW_CTX_DAG_ID=pipeline_try
AIRFLOW_CTX_TASK_ID=thursday_Task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-12T20:57:19.032000+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-12T20:57:19.032000+00:00
[2021-08-12 20:57:27,626] {logging_mixin.py:104} INFO - From thursday_Task, so its thursday today
[2021-08-12 20:57:27,627] {python.py:151} INFO - Done. Returned value was: None
[2021-08-12 20:57:27,641] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=pipeline_try, task_id=thursday_Task, execution_date=20210812T205719, start_date=20210812T205727, end_date=20210812T205727
[2021-08-12 20:57:27,682] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-08-12 20:57:27,690] {local_task_job.py:149} INFO - Task exited with return code 0
